docs/doxygen/Doxymain.md

```diff
@@ -0,0 +1,76 @@
+# Welcome to p4 Compiler's documentation!
+
+@htmlonly
+<div class="diagram-container">
+<!-- draw.io diagram -->
+<div class="mxgraph" style="" data-mxgraph="{&quot;highlight&quot;:&quot;#0000ff&quot;,&quot;lightbox&quot;:false,&quot;nav&quot;:true,&quot;edit&quot;:&quot;_blank&quot;,&quot;url&quot;:&quot;https://drive.google.com/uc?id=16X8Z_4-J8mHQPxuxm9HjWppPMTTXZdZ6&amp;export=download&quot;}"></div>
+<script type="text/javascript" src="https://viewer.diagrams.net/embed2.js?&fetch=https%3A%2F%2Fdrive.google.
+com%2Fuc%3Fid%3D16X8Z_4-J8mHQPxuxm9HjWppPMTTXZdZ6%26export%3Ddownload"></script>
+</div>
+
+<button id="toggle" onclick="animateFlow()">Toggle</button>
+
+<script>
+    function animateFlow() {
+        const svgDocument = document.querySelector('.mxgraph svg');
+        if (svgDocument) {
+            const edges = svgDocument.querySelectorAll('path:not([marker-end])');
+            edges.forEach(edge => {
+                edge.classList.add('flow');
+            });
+            const toggleButton = document.getElementById('toggle');
+            if (toggleButton) {
+                toggleButton.style.display = 'none';
+            }
+        }
+    }
+</script>
+@endhtmlonly
+
+Join the future of programmable networking with **p4c**, a tool that is transforming the way we handle network data. As networks evolve, so does the need for flexible and efficient solutions to manage increasing volumes of data. **p4c** stands at the forefront of this evolution, offering a revolutionary approach to network programming.
```

Comment:
```suggestion
Join the future of programmable networking with **P4C**, a tool that is transforming the way we handle network data. As networks evolve, so does the need for flexible and efficient solutions to manage increasing volumes of data. **P4C** stands at the forefront of this evolution, offering a revolutionary approach to network programming.
```

I would tone this down. Think about this paragraph and what information it actually conveys. After reading it, do you really know what P4C is about? 

https://mihaibudiu.github.io/work/p4-osr17.pdf
Is the paper that describes it, much more neutral in tone.